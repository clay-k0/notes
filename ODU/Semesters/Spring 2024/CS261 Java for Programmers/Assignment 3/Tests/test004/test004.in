4
E 10 F
D 90 E
B 75 C F
A 30 C B
C 30
F 10
