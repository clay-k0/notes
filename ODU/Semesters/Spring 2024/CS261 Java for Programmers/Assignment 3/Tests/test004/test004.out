145:  A:30  B:75  C:30  F:10
